.github/workflows/test_build_deb.yaml

```diff
@@ -19,5 +19,7 @@ jobs:
         path: "android-cuttlefish"
     - name: build debian package
       run: cd android-cuttlefish && debuild -i -us -uc -b
-    - name: install debian package
+    - name: install base debian package
```

Comment:
Will all of our instructions have to be updated to be "install `cuttlefish-base...`" and "install `cuttlefish-common...`" rather than just "install `cuttlefish-common...`"?

I believe this was the reason adelva@ suggested using a metapackage in the design, so the internal packages are named something more descriptive but we don't have to change any of the install instructions.